index.html

```diff
@@ -111,11 +111,11 @@ <h2 class="pop-h2">Keeping track of hundreds of<br> components</h2>
             <li class="skills-up">Ruby </li>
             <li class="skills-up">Css</li>
             <li class="skills-up">html</li>
-            <li class="skills-up">Ruby </li>
-            <li class="skills-up">Css</li>
-            <li class="skills-up">html</li>
+            <li class="skills-up" id="dont-show">Ruby </li>
```

Comment:
Currently, the projects don't seem to be saved inside an object, and you're displaying a popup from your HTML file. I have a couple of suggestions to help improve this:

- [ ] First, it would be great if you could avoid hardcoding the popups in your HTML file. Instead, you can use JavaScript, just as the [project requirements suggest](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md). This way, your project will be more dynamic and user-friendly.

- [ ] Secondly, you might consider using an object to store your projects' information. By doing this, you can easily assign the relevant details to each project card, making it much more organized and manageable.

   ### Project requirements
   ![image](https://github.com/RolandAmini/portfolio-mobile/assets/107952724/612197c7-e4ef-412c-8b41-a30342efa063)

   ### Your hardcoded HTML popup:
   ![image](https://github.com/RolandAmini/portfolio-mobile/assets/107952724/b45c4c5a-535c-4088-a40d-ab4a12090785)